integ-test/src/test/java/org/opensearch/sql/calcite/standalone/CalcitePPLBasicIT.java

```diff
@@ -660,4 +703,523 @@ public void testAllFieldsInTable() throws IOException {
             + "}",
         actual);
   }
+
+  @Ignore
+  @Test
+  public void testDate() {
```

Comment:
Move all functions related tests to a new `CalcitePPLBuiltinFunctionIT`, this one is not for functions ITs.